YoutubeDownloader/Views/Components/DashboardView.xaml

```diff
@@ -11,6 +11,11 @@
     d:DataContext="{d:DesignInstance Type=components:DashboardViewModel}"
     FocusManager.FocusedElement="{Binding ElementName=QueryTextBox}"
     mc:Ignorable="d">
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <s:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" FalseVisibility="Hidden" />
+        </ResourceDictionary>
```

Comment:
Can this resource be moved as a resource inside the `DataGrid`, where it's used? As in, `DataGrid.Resources` instead of `UserControl.Resources`.

Reply:
Unfortunately, it can't. The resource must be outside the `DataGrid`.